manifests/g/GNU/Gforth/0.7.0/GNU.Gforth.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GNU.Gforth
+PackageVersion: 0.7.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallModes:
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent
+  SilentWithProgress: /silent

```

InstallerSwitches not required when the InstallerType is correct, added interactive InstallModes for later use
```suggestion
InstallModes:
- interactive
- silent
- silentWithProgress
```